tests/integration/data_quality_test.py

```diff
@@ -71,17 +71,38 @@ def test_lost_facts_pct(extracted, request):
         assert instance_used_ratio > per_filing_threshold and instance_used_ratio <= 1
 
 
-def test_deduplication(extracted):
+def test_publication_time(extracted):
     table_defs, table_data, _stats = extracted
 
     for table_name, table in table_defs.items():
-        date_cols = ["date"] if table.instant else ["start_date", "end_date"]
-        # we want to make sure that any fact only comes from one filing, so we
-        # don't want to include filing_name in the unique_cols
-        if (df := table_data[table_name]).empty:
-            continue
-        unique_cols = ["entity_id"] + date_cols + table.axes
-        assert not df.reset_index().duplicated(unique_cols).any()
+        assert (
+            table_data[table_name]
+            .reset_index(level="publication_time")
+            .publication_time.notna()
+            .all()
+        )
+
+
+def test_all_data_has_corresponding_id(extracted):
```

Comment:
@zaneselvans FYI I wrote a test to make sure this change does give us the guarantee we want.